e2e/load_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Tetrate
```

Comment:
do we really want to run load test as a part of E2E test? I want to run separately from the existing e2e which runs on all the versions of Envoy (in order to see the ABI compatibility), and instead I want to run this e2e only on the latest Envoy because the latter would be much more time consuming by the time when we add the RPS test. So I would suggest to have another make target and GHA job on the latest Envoy, not as a part of existing e2e.

Reply:
sounds nice and that makes sense :+1:

Reply:
done